src/amo/pages/Addon/index.js

```diff
@@ -600,6 +600,8 @@ export class AddonBase extends React.Component {
             <title>{this.getPageTitle()}</title>
             <link rel="canonical" href={addon.url} />
             <meta name="description" content={this.getPageDescription()} />
+            <meta name="date" content={addon.created} />
+            <meta name="last-modified" content={addon.last_updated} />
```

Comment:
since last_updated can be null would it be better to wrap this in a condition? REF: https://github.com/mozilla/addons-frontend/pull/6632/files#diff-9e6cdc9033004286c61eaefcf496afe2R133

Reply:
I suppose yes, but I don't think `last_updated` can be `null` actually. It seems to be always returned by the API.

@EnTeQuAk can `last_updated` be `null` or empty?

Reply:
As per our IRC discussion, it *might* happen, so let's be defensive here. Thanks @rebmullin, that's an extremely relevant review!